docs/chapters/licensing/index.rst

```diff
@@ -0,0 +1,25 @@
+Licensing
+=========
+
+Each source code file needs to mention what license applies and who owns
+the copyright in the file header. The purpose of this is to make it
+absolutely clear, for both humans and tooling, which license it is that
+applies to that particular file even when the file is moved to another
+project. In order to achieve this, it has been decided that the header
+should include:
```

Comment:
The SWF should contain the result of all our discussions, experience, findings etc. We do not have to indicate what type of experience/event that resulted in the conclusions. The SWF should be, by it's nature, the "current truth". The rationale is important to avoid unnecessary discussions and to support constructive improvements, but we don't need to assert the mandate within the docs.